lighthouse-core/gather/driver.js

```diff
@@ -464,8 +464,6 @@ class Driver {
         });
     }
 
-    log.verbose('Driver', `Installing longtask listener for CPU idle.`);
-    this.evaluateScriptOnLoad(`(${installPerformanceObserver.toString()})()`);
```

Comment:
> We could also land this since it works fine so we can get it in the release, and then do a refactor of waitForLoad later.

sg :) I was blinded by my desire to not have a messy rebase of a local branch that was abusing markAsResolvable for the waitForCustomPerformanceMark 😆